src/lib.rs

```diff
@@ -467,6 +467,8 @@ extern crate alloc;
 
 mod duration;
 pub use duration::Duration;
+/// Alias of [`Duration`]
```

Comment:
This should be formatted as a type, that is, with empty lines around it and not between a bunch of imports.